src/redux/books/booksSlice.js

```diff
@@ -0,0 +1,25 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const booksReducer = createSlice({
```

Comment:
- [OPTIONAL] I highly recommend you to change the name of your Slice to something like `booksSlice` to give it more sense, as it is not the reducer

Reply:
Hello @GedeonTS 

Thank you for your kind feedback. I shall make the changes and ask for a review again.

Regards